common/SC_fftlib.cpp

```diff
@@ -217,6 +216,7 @@ static bool scfft_global_initialization(void) {
     }
     fftwf_free(buffer1);
     fftwf_free(buffer2);
+    // printf("SC FFT global init: FFTW initialised.\n");
```

Comment:
I added this for debugging, so both vDSP and FFTW now have commented out debugging info; I could also remove both.

Reply:
fine to leave it :)